docs/dev/ffi/when-to-use-what-in-the-ffi.md

```diff
@@ -15,8 +15,8 @@ there are two kinds of values you may want to pass over the FFI.
 
 ## Types with identity
 
-Examples of this are things like database connections, the FirefoxAccounts
-struct, etc. These types are complex, implemented in rust, and it's not
+Examples of this are things like database connections, the `FirefoxAccounts`
+struct, etc. These types are complex, implemented in Rust, and it's not
```

Comment:
Hah, we might want to change this to name our own experience.

Reply:
Yeah -- but I guess as a follow-on bug...